pyproject.toml

```diff
@@ -97,14 +96,14 @@ line-length = 79
 [tool.ruff]
 line-length = 79
 exclude = ["__init__.py","build",".eggs"]
-select = ["I", "E", "F", "TCH", "TID252"]
+lint.select = ["I", "E", "F", "TCH", "TID252"]
 fix = true
-ignore = ["E203","E501","E731","C901","W291","W293","E402","E722"]
+lint.ignore = ["E203","E501","E731","C901","W291","W293","E402","E722"]
 
-[tool.ruff.per-file-ignores]
+[tool.ruff.lint.per-file-ignores]
 "__init__.py" = ["F401"]
 
-[tool.ruff.mccabe]
+[tool.ruff.lint.mccabe]
 max-complexity = 18
 
 [tool.ruff.lint.flake8-tidy-imports]
```

Comment:
I might be mistaken, but I think you can have a `[tool.ruff.lint]` section, under which you can group all setting that start with `lint`.

Reply:
Thanks I changed the main header to `[tool.ruff.lint]` and removed `lint` from the entries within it and it worked. It may be also possible to add the other sections like this also, but I could not see example in the docs e.g. for `mccabe`. But don't have time to test ATM but will keep an eye on this in future!

Reply:
Oh wait I see what you mean now, thanks!